src/events/aws_sdk.js

```diff
@@ -923,9 +926,35 @@ const SSMEventCreator = {
         switch (response.request.operation) {
         case 'getParameter':
             eventInterface.addToMetadata(event, {
-                value: (response.data.Parameter || { Value: '' }).Value || '',
+                parameter: {
+                    Name: (response.data.Parameter || { Name: '' }).Name || '',
+                    Value: (response.data.Parameter || { Value: '' }).Value || '',
+                    Type: (response.data.Parameter || { Type: '' }).Type || '',
+                },
             });
             break;
+
+        case 'getParameters':
+        case 'getParametersByPath':
+            if (response.data.InvalidParameters && response.data.InvalidParameters.length > 0) {
+                eventInterface.addToMetadata(
+                    event,
+                    { invalid_parameters: response.data.InvalidParameters }
+                );
+            }
+
+            if (response.data.Parameters && response.data.Parameters.length > 0) {
+                eventInterface.addToMetadata(event, {
+                    parameters: response.data.Parameters
```

Comment:
maybe add it as a second dict to `addToMetadata` function, since this probably shouldn't be sent when metadataOnly = True (@idonava would love if you can give an osher a brief of `metadataOnly` mode)

Reply:
fixed

Reply:
does this mean, that secret/encrypted parameters from the SSM are stored in the traces?

Reply:
Yes, but you can omit it with with the ignoredKeys (add parameters, or any key inside)